demand_acep/extract_data_to_csv.py

```diff
@@ -76,23 +84,42 @@ def extract_csv_for_date(config, data_date):
                 csv_name = os.path.join(data_path, meter_csv_names[meter])
                 # Only extract if not already pickled
                 if (not os.path.isfile(csv_name)):
-                    # print(meter)
-                    [channel_time, channel_values] = extract_data(dirpath, filename)
+                    channel_df = extract_data(dirpath, filename)
+                    channel_df.columns = [channel]
+                    channel_resampled = data_resample(channel_df, sample_time)
                     if meter_collection[meter].empty:
-                        # meter_collection[meter] = meter_collection[meter].append({'time': channel_time}, ignore_index=True)
-                        # meter_collection[meter].loc[:, channel] = channel_values
-                        meter_collection[meter]['time'] = channel_time
-                    meter_collection[meter][channel] = channel_values
+                        meter_collection[meter] = channel_resampled.copy()
+                    else:
+                        if channel in meter_collection[meter].columns:
+                            idx_1 = meter_collection[meter].index
+                            idx_2 = channel_resampled.index
+                            if np.all(np.isin(idx_2, idx_1)):
+                                meter_collection[meter][channel].loc[idx_2] = np.reshape(channel_resampled.values,
+                                                                            (len(channel_resampled),))
+                            else:
+                                meter_collection[meter] = meter_collection[meter].append(channel_resampled, sort=False)
+                                meter_collection[meter].sort_index(inplace=True)
+                                meter_collection[meter] = data_resample(meter_collection[meter], sample_time)
```

Comment:
@aidowu Is the re-sampling twice here?. I thought we were already re-sampling in line 89.